README.md

```diff
@@ -5,9 +5,11 @@
 
```

Comment:
this is awesome! If you plan on continuing development on this maybe I can make you an owner and ill just shadow your commits to make sure its all kosher. This means every push to master should be stable and working though!